folder_paths.py

```diff
@@ -78,6 +78,14 @@ def get_input_directory() -> str:
     global input_directory
     return input_directory
 
+def get_user_directory() -> str:
+    global user_directory
```

Comment:
https://stackoverflow.com/questions/11867359/when-do-i-need-to-use-the-global-keyword-in-python

Reply:
Thanks! Removed global.